man_nopan is for !USE_PAM already included in man_MANS and automake-1.11 hates to install some file twice

diff -uNr shadow-4.1.4.2.orig/man/Makefile.am shadow-4.1.4.2/man/Makefile.am
--- shadow-4.1.4.2.orig/man/Makefile.am	2009-03-14 15:40:10.000000000 +0100
+++ shadow-4.1.4.2/man/Makefile.am	2010-04-02 07:31:17.000000000 +0200
@@ -163,7 +163,6 @@
 	$(man_MANS) \
 	$(man_XMANS) \
 	$(addprefix login.defs.d/,$(login_defs_v)) \
-	$(man_nopam) \
 	id.1 \
 	id.1.xml \
 	sulogin.8 \
diff -uNr shadow-4.1.4.2.orig/man/fr/Makefile.am shadow-4.1.4.2/man/fr/Makefile.am
--- shadow-4.1.4.2.orig/man/fr/Makefile.am	2008-09-06 18:44:45.000000000 +0200
+++ shadow-4.1.4.2/man/fr/Makefile.am	2010-04-02 07:42:11.000000000 +0200
@@ -52,7 +52,6 @@
 
 EXTRA_DIST = \
 	$(man_MANS) \
-	$(man_nopam) \
 	id.1
 
 include ../generate_translations.mak
diff -uNr shadow-4.1.4.2.orig/man/it/Makefile.am shadow-4.1.4.2/man/it/Makefile.am
--- shadow-4.1.4.2.orig/man/it/Makefile.am	2008-09-06 18:44:45.000000000 +0200
+++ shadow-4.1.4.2/man/it/Makefile.am	2010-04-02 07:42:20.000000000 +0200
@@ -46,7 +46,6 @@
                         
 EXTRA_DIST = \
 	$(man_MANS) \
-	$(man_nopam) \
 	id.1 \
 	logoutd.8
 
diff -uNr shadow-4.1.4.2.orig/man/ja/Makefile.am shadow-4.1.4.2/man/ja/Makefile.am
--- shadow-4.1.4.2.orig/man/ja/Makefile.am	2007-12-31 17:48:28.000000000 +0100
+++ shadow-4.1.4.2/man/ja/Makefile.am	2010-04-02 07:42:17.000000000 +0200
@@ -49,7 +49,6 @@
 
 EXTRA_DIST = \
 	$(man_MANS) \
-	$(man_nopam) \
 	id.1 \
 	shadow.3 \
 	sulogin.8
diff -uNr shadow-4.1.4.2.orig/man/pl/Makefile.am shadow-4.1.4.2/man/pl/Makefile.am
--- shadow-4.1.4.2.orig/man/pl/Makefile.am	2008-09-06 18:44:45.000000000 +0200
+++ shadow-4.1.4.2/man/pl/Makefile.am	2010-04-02 07:42:07.000000000 +0200
@@ -49,7 +49,6 @@
 
 EXTRA_DIST = \
 	$(man_MANS) \
-	$(man_nopam) \
 	getspnam.3 \
 	id.1 \
 	shadow.3 \
diff -uNr shadow-4.1.4.2.orig/man/ru/Makefile.am shadow-4.1.4.2/man/ru/Makefile.am
--- shadow-4.1.4.2.orig/man/ru/Makefile.am	2010-04-02 07:39:00.000000000 +0200
+++ shadow-4.1.4.2/man/ru/Makefile.am	2010-04-02 07:42:01.000000000 +0200
@@ -54,7 +54,6 @@
 
 EXTRA_DIST = \
 	$(man_MANS) \
-	$(man_nopam) \
 	id.1 \
 	sulogin.8
 
diff -uNr shadow-4.1.4.2.orig/man/sv/Makefile.am shadow-4.1.4.2/man/sv/Makefile.am
--- shadow-4.1.4.2.orig/man/sv/Makefile.am	2008-09-06 18:44:45.000000000 +0200
+++ shadow-4.1.4.2/man/sv/Makefile.am	2010-04-02 07:42:24.000000000 +0200
@@ -53,8 +53,7 @@
 endif
 
 EXTRA_DIST = \
-	$(man_MANS) \
-	$(man_nopam)
+	$(man_MANS)
 
 include ../generate_translations.mak
 
--- shadow-4.1.4.2.orig/man/ru/Makefile.am	2010-04-02 07:54:09.000000000 +0200
+++ shadow-4.1.4.2/man/ru/Makefile.am	2010-04-02 07:51:57.000000000 +0200
@@ -1,7 +1,6 @@
 mandir = @mandir@/ru
 
 man_MANS = \
-	$(man_nopam) \
 	chage.1 \
 	chfn.1 \
 	chgpasswd.8 \
